### PR TITLE
update github actions

### PR DESCRIPTION
**Pull request by romainfrancois**

### CODE REVIEW

**Comment by romainfrancois** [comment]
closes #114

**Comment by romainfrancois** [comment]
I tried to make it depend on R 3.4 as in #114, but there seems to be an issue building `stringfish`. cc @traversc 

https://github.com/tidyverse/multidplyr/runs/4368543258?check_suite_focus=true#step:5:378

File: README.Rmd [hunk 1]
```diff
@@ -28,8 +28,6 @@ multidplyr is a backend for dplyr that partitions a data frame across multiple c
 
 Due to the overhead associated with communicating between the nodes, you won't see much performance improvement with simple operations on less than ~10 million observations, and you may want to instead try [dtplyr](https://dtplyr.tidyverse.org/), which uses [data.table](https://R-datatable.com/). multidplyr's strength is found parallelising calls to slower and more complex functions.
 
-(Note that unlike other packages in the tidyverse, multidplyr requires R 3.5 or greater. We hope to relax this requirement [in the future](https://github.com/traversc/qs/issues/11).)
```
Comment (hadley) [inline README.Rmd:31]: Did you mean to delete this?

**Comment by traversc** [comment]
Sorry, it should be fixed now (again). https://github.com/traversc/stringfish/runs/4377875621?check_suite_focus=true#step:7:120

There was a simple macro issue where I forgot to define a function in R < 3.5. I thought I tested R 3.4, but at some point oldrel-3 changed to R 3.5 :P

File: README.Rmd (hunk 1 above)
Comment (romainfrancois) [reply README.Rmd:31]: oops. I did, but then I didn't.

**Comment by romainfrancois** [comment]
Thanks @traversc, will you release `stringfish` to cran ?

**Comment by traversc** [comment]
Updated on CRAN.

**Comment by hadley** [comment]
Thanks @traversc, added in #130.

**Pull request merged**